src/MasterPeace/Bundle/BookBundle/DataFixtures/ORM/LoadBookData.php

```diff
@@ -23,8 +26,14 @@ public function load(ObjectManager $manager)
                 ->setCover($bookDetail['cover'])
                 ->setIsbnCode($bookDetail['isbn_code']);
             $manager->persist($book);
+
+            $bookObject->append($book);
         }
         $manager->flush();
+
+        foreach ($bookObject as $id => $book) {
+            $this->addReference('book' . $id, $book);
```

Comment:
as is pradziu dejau i to bookDetail foreach tai jis ismesdavo errora. Kaip supratau reik visus sukurt ir poto tik naudot addreference? Nu dar gryzes namo pabandysiu paziet :)